include/idawrap.h

```diff
@@ -130,8 +131,21 @@ class IDASolver : public Solver
         return(0);
     }
 
-    int AMISetId(N_Vector id) {
-        return IDASetId(ami_mem, id);
+    int AMISetId(Model *model) {
```

Comment:
id should be a constant vector that is set at model initialisation. It must not be modified by the user.